commit 22b564b0b6909664bd7adac2fa9f3f0abc46457a
Author: Jason Cooper <jason@lakedaemon.net>
Date:   Sat Oct 9 14:51:15 2010 -0400

    staging: brcm80211: remove BLOCKABLE so no one uses it.
    
    BLOCKABLE() doesn't do what it's supposed to.
    
    Signed-off-by: Jason Cooper <jason@lakedaemon.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/brcmfmac/dhd_linux.c
+++ b/drivers/staging/brcm80211/brcmfmac/dhd_linux.c
@@ -333,8 +333,6 @@ uint dhd_radio_up = 1;
 char iface_name[IFNAMSIZ];
 module_param_string(iface_name, iface_name, IFNAMSIZ, 0);
 
-#define BLOCKABLE()	(!in_atomic())
-
 /* The following are specific to the SDIO dongle */
 
 /* IOCTL response timeout */
